src/builtins/operators.md

```diff
@@ -0,0 +1,118 @@
+#### Built-in Operators
+
+As [operators](../operators.md) in Alan are simply aliases for functions with a precedence value associated, all of the following operators have their underlying implementation defined above.
+
+<table>
+  <thead>
+    <tr>
+      <td>Operator</td><td>Infix/Prefix</td><td>Precedence</td><td>Function</td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>?</code></td><td>infix</td><td>0</td><td>cond</td>
```

Comment:
Now that this table is on a different page, putting a link to the function and/or the function description here is probably wise.